WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/blaze/BlazeCampaignsCardView.kt

```diff
@@ -1,93 +1,216 @@
-            width = dimensionResource(id = R.dimen.unelevated_card_stroke_width),
-            color = colorResource(id = R.color.on_surface_divider)
-        ),
-        elevation = 0.dp,
-        shape = RoundedCornerShape(10.dp)
-    ) {
-        Column(
+    DashboardCard(modifier = modifier, content = {
+        Text(
+            text = uiStringText(uiString = blazeCampaignCard.title),
+            style = smallTitle(),
+            textAlign = TextAlign.Start,
             modifier = modifier
                 .fillMaxWidth()
-                .fillMaxHeight()
-        ) {
-            Text(
-                text = uiStringText(uiString = blazeCampaignCard.title),
-                fontSize = 14.sp,
-                textAlign = TextAlign.Start,
-                color = colorResource(R.color.material_on_surface_emphasis_high_type),
-                modifier = modifier
-                    .fillMaxWidth()
-                    .wrapContentHeight()
-                    .padding(start = 16.dp, top = 16.dp)
-            )
-            Text(
-                text = uiStringText(uiString = blazeCampaignCard.campaign.status),
-                fontSize = 12.sp,
-                fontWeight = FontWeight.Normal,
-                color = colorResource(R.color.material_on_surface_emphasis_medium),
-                textAlign = TextAlign.Start,
-                modifier = modifier
-                    .fillMaxWidth()
-                    .wrapContentHeight()
-                    .padding(start = 16.dp, top = 8.dp)
+                .wrapContentHeight()
+                .padding(start = 16.dp, top = 16.dp)
+        )
+        val status = blazeCampaignCard.campaign.status
+        if (status != null) {
+            BlazeStatusLabel(
+                status = status,
+                isInDarkMode = isInDarkMode
             )
-            Text(
-                text = uiStringText(uiString = blazeCampaignCard.campaign.title),
-                fontSize = 16.sp,
-                fontWeight = FontWeight.Normal,
-                textAlign = TextAlign.Start,
-                color = colorResource(R.color.material_on_surface_emphasis_medium),
-                modifier = modifier
-                    .fillMaxWidth()
+        }
+        CampaignTitleThumbnail(
+            campaignTitle = blazeCampaignCard.campaign.title,
+            featuredImageUrl = blazeCampaignCard.campaign.featuredImageUrl,
+            modifier = Modifier
+                .fillMaxWidth()
+                .wrapContentHeight()
+        )
+        if (blazeCampaignCard.campaign.stats != null) {
+            CampaignStats(
+                campaignStats = blazeCampaignCard.campaign.stats,
+                modifier = Modifier
+                    .wrapContentWidth()
                     .wrapContentHeight()
-                    .padding(start = 16.dp, top = 8.dp)
+                    .background(Color.LightGray)
+                    .padding(start = 16.dp, top = 8.dp, bottom = 8.dp)
```

Comment:
💡 This `top` padding is not the usual way of doing spacing between items in Compose. It's far more common to use [`Spacer`](https://developer.android.com/reference/kotlin/androidx/compose/foundation/layout/package-summary#Spacer(androidx.compose.ui.Modifier)) (with `Modifier.width()` for Rows, and `Modifier.height()` for columns) or, if all spacing between items in those containers is equal, you can play with [`Arrangement`](https://developer.android.com/reference/kotlin/androidx/compose/foundation/layout/Arrangement): `horizontalArrangement` (for rows) or `verticalArrangement` (for columns) with `Arrangement.spacedBy()` and `Arrangemement.SpaceBetween`.

The `padding()` modifier is mostly used for internal paddings inside components or around children in container layouts like Row and Column, so the equivalent of View-based `padding` and not `margin`.

As a rule of thumb, you can think about it this way: 
- if you're trying to achieve something that would be a `margin` between components in an XML layout, you probably want a `Spacer` or `Arrangement`.
- if you want to try to achieve what would be a `padding` inside a component in an XML layout or a `margin` **around** your component, you probably want a `padding()` modifier.

Overall that "margin around component" scenario is not very common since that can be done as a `padding` inside the parent container composable, which is usually a Row, Column, or Box, once you get used to thinking in Compose.